lib/templates/namespaces.html

```diff
@@ -0,0 +1,200 @@
+            </path>
+          </svg>
+        </a>
+      </div>
+      <div class="govuk-header__content">
+        <h1 href="#" class="govuk-header__link govuk-header__service-name">
+          Cloud Platform Reports: Costs by Namespace
+        </h1>
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav mr-auto">
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true"
+                  aria-expanded="false">Todo</a>
+                <div class="dropdown-menu">
+                  <a class="dropdown-item" href="/dashboard">Dashboard</a>
+                  <a class="dropdown-item" href="/helm_whatup">Helm Releases</a>
+                  <a class="dropdown-item" href="/terraform_modules">Terraform Modules</a>
+                  <a class="dropdown-item" href="/documentation">Documentation</a>
+                  <a class="dropdown-item" href="/orphaned_resources">Orphaned AWS Resources</a>
+                  <a class="dropdown-item" href="/orphaned_statefiles">Orphaned Terraform Statefiles</a>
+                </div>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true"
+                  aria-expanded="false">Reports</a>
+                <div class="dropdown-menu">
+                  <a class="dropdown-item" href="/costs_by_namespace">Costs by Namespace</a>
+                  <a class="dropdown-item" href="/hosted_services">Hosted Services</a>
+                  <a class="dropdown-item" href="/namespace_usage">Namespace Resource Usage</a>
+                  <a class="dropdown-item" href="/live_1_domains">Services with live-1 Domains</a>
+                  <a class="dropdown-item" href="/infrastructure_deployments">Infrastructure Deployments</a>
+                </div>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="/about">About</a>
+              </li>
+            </ul>
+            <ul class="navbar-nav justify-content-end">
+              <li class="nav-item">
+                <a class="nav-link"
+                  href="https://github.com/ministryofjustice/cloud-platform-how-out-of-date-are-we">GitHub</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
+    </div>
+  </header>
+  <div class="container-fluid">
+    <h2 class="page_heading">Summary</h2>
```

Comment:
Can we change this to Namespace and then add the Namespace name underneath so that it's properly visible please